spa/src/components/Error/KnownErrors/index.tsx

```diff
@@ -1,54 +1,55 @@
+import { css } from "@emotion/react";
 import { token } from "@atlaskit/tokens";
 import Button from "@atlaskit/button";
 import analyticsClient from "../../../analytics";
 import { popup } from "../../../utils";
 
-const Paragraph = styled.div`
+const paragraphStyle = css`
 	color: ${token("color.text.subtle")};
 `;
-const BulletSeparator = styled.span`
+const bulletSeparatorStyle = css`
 	padding: 0 ${token("space.100")};
 `;
-const StyledLink = styled.a`
+const linkStyle = css`
 	cursor: pointer;
 `;
 
 /************************************************************************
  * UI view for the 3 known errors
  ************************************************************************/
 export const ErrorForSSO = ({ orgName, accessUrl, resetCallback }: { orgName?: string; accessUrl: string; resetCallback: () => void;}) => <>
-	<Paragraph>
+	<div css={paragraphStyle}>
 		Can't connect, single sign-on(SSO) required{orgName && <span> for <b>{orgName}</b></span>}.
-	</Paragraph>
-	<Paragraph>
-		1. <StyledLink onClick={() => popup(accessUrl)}>Log into GitHub with SSO</StyledLink>.
-	</Paragraph>
-	<Paragraph>
-		2. <StyledLink onClick={resetCallback}>Retry connection in Jira</StyledLink> (once logged in).
-	</Paragraph>
+	</div>
+	<div css={paragraphStyle}>
+		1. <a css={linkStyle} onClick={() => popup(accessUrl)}>Log into GitHub with SSO</a>.
+	</div>
+	<div css={paragraphStyle}>
+		2. <a css={linkStyle} onClick={resetCallback}>Retry connection in Jira</a> (once logged in).
+	</div>
 </>;
 
-export const ErrorForNonAdmins = ({ orgName, adminOrgsUrl }: { orgName?: string; adminOrgsUrl: string; }) => <Paragraph>
+export const ErrorForNonAdmins = ({ orgName, adminOrgsUrl }: { orgName?: string; adminOrgsUrl: string; }) => <div css={paragraphStyle}>
 	Can't connect, you're not the organization owner{orgName && <span> of <b>{orgName}</b></span>}.<br />
-	Ask an <StyledLink onClick={() => {
+	Ask an <a css={linkStyle} onClick={() => {
 	// TODO: Need to get this URL for Enterprise users too, this is only for Cloud users
 		popup(adminOrgsUrl);
 		analyticsClient.sendUIEvent({ actionSubject: "checkOrgAdmin", action: "clicked"}, { type: "cloud" });
-	}}>organization owner</StyledLink> to complete this step.
-</Paragraph>;
+	}}>organization owner</a> to complete this step.
+</div>;
 
 export const ErrorForIPBlocked = ({ orgName, resetCallback }: { orgName?: string; resetCallback: () => void }) => <>
-	<Paragraph>
+	<div css={paragraphStyle}>
 		Can't connect{orgName && <span> to <b>{orgName}</b></span>}, blocked by your IP allow list.
-	</Paragraph>
+	</div>
 	<Button
 		style={{ paddingLeft: 0, paddingRight: 0 }}
 		appearance="link"
 		onClick={() => popup("https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md")}
 	>
 		How to update allowlist
 	</Button>
-	<BulletSeparator>&#8226;</BulletSeparator>
-	<StyledLink onClick={resetCallback}>Retry</StyledLink>
+	<span css={bulletSeparatorStyle}>&#8226;</span>
```

Comment:
If you'd like, you can actually just use the bullet character directly "•" and that'll render fine :)

I'm curious what this looks like on the UI though. Are we using a bullet character for one item? Or is this a list (and if it is, we should probably be using a `<ul>`?

Reply:
Its not an actual list or anything, just a dot separator for 2 separate sections.